src/relayserver/RelayServer.ts

```diff
@@ -417,72 +291,81 @@ export class RelayServer extends EventEmitter {
   }
 
   async _init (): Promise<void> {
+    // TODO: add rerun protection
```

Comment:
`if (this.initialized) {
      return
  }`

Reply:
fixed